lib/components/Users/UserRoutes.ts

```diff
@@ -99,6 +98,60 @@ export function login(req: NextApiRequest, res: NextApiResponse) {
 
 
 
+export function verifyUserToken(req, res) {
```

Comment:
Manually verifiy the jwt token after each session. The manual verification status in the front-end gets reset each time the user leaves/closes the window, so when they come back, it'll be manually verified to make sure their session hasn't expired